### PR TITLE
Update ROS Instructions

### LINKED ISSUE #660: Improve ROS Docs
Now that our ros packages are official and can be installed with apt-get we should simplify the ros installation instructions.

### PR DESCRIPTION
**Pull request by DavidMansolino**
Fix #660
Preview:
  - https://cyberbotics.com/doc/guide/using-ros?version=fix-ros-instruction
  - https://cyberbotics.com/doc/guide/tutorial-8-using-ros?version=fix-ros-instruction

This PR also include modifications on the ros wiki:
   - http://wiki.ros.org/webots_ros
   - http://wiki.ros.org/webots_ros/Tutorials/Sample%20Simulations

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
:eyes:

**Comment by stefaniapedrazzi** [comment]
The new webots-ros package version 2.0.4 is not available yet for melodic.
I will test again when it will be ready.

**Comment by DavidMansolino** [comment]
> The new webots-ros package version 2.0.4 is not available yet for melodic.
I will test again when it will be ready.

Yes indeed, let's put this PR on hold, I will put the thumbs-up again when it will be released.

**Comment by DavidMansolino** [comment]
The packages are now released, I will try them and if everything is working fine I will add the thumb-up again.

**Comment by DavidMansolino** [comment]
:+1:

File: docs/guide/using-ros.md [hunk 1]
```diff
@@ -51,9 +52,9 @@ For services, it is also the name of the corresponding srv file.
 The controller, named `ros`, is pre-compiled and you shouldn't edit it.
 All you have to do is to load it in the `controller` field of your robot; you will find it in the default list of controller.
 In order to use it, you will have to build a ROS node that will communicate with the robot using the different services available.
-Good examples of such ROS node can be found inside Webots at "[WEBOTS\_HOME/projects/languages/ros/webots\_ros](https://github.com/omichel/webots/tree/master/projects/languages/ros/webots_ros)".
+Good examples of such ROS node can be found inside Webots at "[WEBOTS\_HOME/projects/languages/ros/webots\_ros](https://github.com/omichel/webots/tree/master/projects/languages/ros/webots_ros)", they are documented in the package [webots\_ros package tutorial](http://wiki.ros.org/webots_ros/Tutorials/Sample%20Simulations).
```
Comment (omichel) [inline docs/guide/using-ros.md]: I would remove the first instance of "package" here.

**Pull request merged**